qiskit_nature/properties/second_quantization/electronic/integrals/electronic_integrals.py

```diff
@@ -97,10 +109,31 @@ def __str__(self) -> str:
     def _render_matrix_as_sparse_list(matrix) -> List[str]:
         string = []
         nonzero = matrix.nonzero()
+        nonzero_count = len(nonzero[0])
+        string += [f"\t<{matrix.shape} matrix with {nonzero_count} non-zero entries>"]
+        count = 0
         for value, *indices in zip(matrix[nonzero], *nonzero):
+            if ElectronicIntegrals._truncate and count >= ElectronicIntegrals._truncate:
+                string += [
+                    f"\t... skipping {nonzero_count - ElectronicIntegrals._truncate} entries"
+                ]
+                break
             string += [f"\t{indices} = {value}"]
+            count += 1
         return string
 
+    @staticmethod
+    def set_truncation(max_num_entries: int) -> None:
+        """Set the maximum number of integral values to display before truncation.
+
+        Args:
+            max_num_entries: the maximum number of entries.
+
+        .. note::
+            Truncation will be disabled if `max_num)lines` is set to 0.
```

Comment:
```suggestion
            Truncation will be disabled if `max_num_entries` is set to 0.
```
I imagine this is what it should be

Reply:
Yes you're right! I had originally used `max_num_lines` as the argument name but apparently I even misspelled that :sweat_smile: